app/models/miq_request_workflow.rb

```diff
@@ -1502,13 +1502,15 @@ def ws_schedule_fields(values, _fields, data)
   end
 
   def validate_values(values)
-    if validate(values) == false
-      errors = []
-      fields { |_fn, f, _dn, _d| errors << f[:error] unless f[:error].nil? }
-      err_text = "Provision failed for the following reasons:\n#{errors.join("\n")}"
-      _log.error "<#{err_text}>"
-      raise _("Provision failed for the following reasons:\n%{errors}") % {:errors => errors.join("\n")}
-    end
+    raise_validate_errors if validate(values) == false
+  end
```

Comment:
@lfu Is the `validate_values` method still needed?  I do not see anyone calling it.